project1/server/file_handler/file_handler.h

```diff
@@ -21,23 +22,29 @@
 namespace server::file_handler {
 class FileHandler : public IFileHandler {
  public:
+  FileHandler();
+
   [[nodiscard]] std::vector<uint8_t> Get(
-      const std::string& filename) const final;
+      const std::string& filename) const override;
```

Comment:
I am probably missing something here, but why did these go from `final` to `override`?

Reply:
`final` actually prevents you from overriding them in a subclass. `override` does not. I usually prefer `final` and only fall back on the more permissive `override` when it's actually necessary.